webapp/src/ts/services/browser-detector.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+import * as Bowser from 'bowser';
+import { Store } from '@ngrx/store';
+import { Selectors } from '@mm-selectors/index';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BrowserDetectorService {
+  private _parser: Bowser.Parser.Parser;
+  private androidAppVersion: VersionNumber | undefined;
+
+  public constructor(private store: Store) {
+    this.store.select(Selectors.getAndroidAppVersion).subscribe(androidAppVersion => {
+      this.androidAppVersion = androidAppVersion;
+    });
+  }
+
+  private get parser() {
+    // lazy initialization to allow unit tests to use a different user agent
+    if (typeof this._parser === 'undefined') {
+      this._parser = Bowser.getParser(window.navigator.userAgent);
+    }
+
+    return this._parser;
+  }
+
+  public detect() {
+    return this.parser.getBrowser();
+  }
+
+  public isUsingSupportedBrowser() {
+    return this.parser.satisfies({
+      chrome: '>=53',
```

Comment:
I've thought about this but couldn't come up with a solution in a reasonable amount of time. What would be even better is if it could be shared with the docs in `cht-docs` https://docs.communityhealthtoolkit.org/core/releases/#dependencies